javaBox.js

```diff
@@ -5,81 +5,42 @@ const Docker = require('dockerode');
 const docker = new Docker();
 
 // Time to wait between checking that the command has been executed (milliseconds)
-const EXEC_WAIT_TIME = 100;
+const EXEC_WAIT_TIME = 250;
```

Comment:
Specify the unit of measure in the name of the variable, e.g., `EXEC_WAIT_TIME_MS`